### PR TITLE
fix hashing of RawModules

### PR DESCRIPTION
**Pull request by jbellenger**
webpack-2 appears to have a [bug](https://github.com/webpack/webpack/issues/5070) where RawModules can produce hash collisions. Since globalize-webpack-plugin-1.0.0 introduced use of RawModules, this package is susceptible to emitting chunks with different contents but unchanged chunkhashes. This can cause cdns and browser caches to unwittingly use old assets and lead to irksome bugs.

Even though the root issue appears to be in webpack, this package should be patched to work around the issue in the versions of webpack that it supports.

Additionally, the required webpack version here is downgraded to match what is required by react-globalize-webpack-plugin.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "globalize-webpack-plugin",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (rxaviers) [inline package.json]: Please, let do the version bump in a separate commit

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```
Comment (rxaviers) [inline package.json]: > Additionally, the required webpack version here is downgraded to match what is required by react-globalize-webpack-plugin.

I don't get it... If the consumer product is using webpack 2.5.1 it would satisfy both GWP and RGWP, wouldn't it?

Comment (jbellenger) [inline package.json]: This is mostly to support webpack 2.4.1, which we're using at twitter due to https://github.com/webpack/webpack/issues/4850 in 2.5.1 and above

File: package.json (hunk 1 above)
Comment (jbellenger) [inline package.json]: can do, will update

**Comment by jbellenger** [comment]
Amended diff to not bump package version.

File: package.json (hunk 2 above)
Comment (rxaviers) [inline package.json]: What about the bug fixed in 2.5.1 (i.e., https://github.com/webpack/webpack/pull/4815)? Wouldn't the downgrade allow that to happen again?

Comment (jbellenger) [inline package.json]: Funny that you should mention it, but the reason we're on 2.4.1 at twitter is that  webpack/webpack#4815 introduced webpack/webpack#4850. There is a [workaround](https://github.com/webpack/webpack/issues/4795#issuecomment-302824613) for the issue that works in 2.4, but not 2.5 and above.

Regardless, every project that uses GWP with webpack is responsible for finding the right mix of versions that works for them; I don't know if GWP needs to have an opinion here outside of the specific things that it needs from webpack. GWP does not have anything in particular that depends on 2.5.1.

Comment (rxaviers) [inline package.json]: Agreed. One last confirmation, after this change, would GWP work for someone using webpack 2.5.1?

Comment (jbellenger) [inline package.json]: It would. The user would have more options to manage their webpack version without GWP complaining about bad peer dependencies. Unless a specific webpack version is specified, running `yarn add "webpack@^2.2.0-rc"` will install 2.6.1

Comment (rxaviers) [inline package.json]: Thanks
